docker/development-easy-arm/docker-compose.yml

```diff
@@ -1,107 +1,85 @@
-      OPENEMR_SETTING_couchdb_pass: password
-      OPENEMR_SETTING_couchdb_dbase: example
-      OPENEMR_SETTING_couchdb_ssl_allow_selfsigned: 1
-      OPENEMR_SETTING_user_debug: 3
-      OPENEMR_SETTING_gbl_ldap_host: 'ldap://openldap:389'
-      OPENEMR_SETTING_gbl_ldap_dn: 'cn={login},dc=example,dc=org'
-    depends_on:
-    - mysql
-  phpmyadmin:
-    restart: always
-    image: phpmyadmin/phpmyadmin
-    ports:
-    - 8310:80
-    environment:
-      PMA_HOSTS: mysql
-  couchdb:
-    restart: always
-    image: couchdb
-    ports:
-    - 5984:5984
-    - 6984:6984
-    volumes:
-    - ../development-insane/couchdb-config-ssl-cert-keys/local.ini:/opt/couchdb/etc/local.ini:rw
-    - ../development-insane/couchdb-config-ssl-cert-keys/easy/ca.pem:/etc/ssl/ca.pem:ro
-    - ../development-insane/couchdb-config-ssl-cert-keys/easy/server-cert.pem:/etc/ssl/server-cert.pem:ro
-    - ../development-insane/couchdb-config-ssl-cert-keys/easy/server-key.pem:/etc/ssl/server-key.pem:ro
-    - couchdbvolume:/opt/couchdb/data
-    environment:
-      COUCHDB_USER: admin
-      COUCHDB_PASSWORD: password
-  #maybe later
-  #openldap:
-  #  restart: always
-  #  image: openemr/dev-ldap:easy
-  #  environment:
-  #    LDAP_TLS_VERIFY_CLIENT: try
-  #    LDAP_TLS_CA_CRT_FILENAME: ca.pem
-  #    LDAP_TLS_CRT_FILENAME: server-cert.pem
-  #    LDAP_TLS_KEY_FILENAME: server-key.pem
-volumes:
-  databasevolume: {}
-  publicvolume: {}
-  sitesvolume: {}
-  nodemodules: {}
-  vendordir: {}
-  ccdamodule: {}
-  logvolume: {}
-  couchdbvolume: {}
+version: '3.1'
+services:
+  mysql:
+    restart: always
+    image: jsurf/rpi-mariadb
+    command: ['mysqld','--character-set-server=utf8mb4','--ssl-ca=/etc/ssl/ca.pem','--ssl_cert=/etc/ssl/server-cert.pem','--ssl_key=/etc/ssl/server-key.pem']
+    ports:
+    - 8320:3306
+    volumes:
+    - ../development-insane/sql-ssl-certs-keys/easy/ca.pem:/etc/ssl/ca.pem:ro
+    - ../development-insane/sql-ssl-certs-keys/easy/server-cert.pem:/etc/ssl/server-cert.pem:ro
+    - ../development-insane/sql-ssl-certs-keys/easy/server-key.pem:/etc/ssl/server-key.pem:ro
+    - databasevolume:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+  openemr:
+    restart: always
+    image: openemr/openemr:flex
+    ports:
+    - 8300:80
+    - 9300:443
+    volumes:
+    - ../..:/openemr:ro
+    - ../..:/var/www/localhost/htdocs/openemr:rw
+    - publicvolume:/var/www/localhost/htdocs/openemr/public:rw
+    - sitesvolume:/var/www/localhost/htdocs/openemr/sites:rw
+    - nodemodules:/var/www/localhost/htdocs/openemr/node_modules:rw
+    - vendordir:/var/www/localhost/htdocs/openemr/vendor:rw
+    - ccdamodule:/var/www/localhost/htdocs/openemr/ccdaservice:rw
+    - logvolume:/var/log
+    environment:
+      DEBUG_COLORS: "true"
+      TERM: xterm-256color
+      COLORTERM: truecolor
+      MYSQL_HOST: mysql
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_USER: openemr
+      MYSQL_PASSWORD: openemr

```

Comment:
why was this line modified?

Reply:
The mariadb version that this uses has that variable set to PASSWORD instead of PASS. This might need to be forwarded to the docker differently but I'm not sure.

Reply:
see above